tap_mssql/sync_strategies/log_based.py

```diff
@@ -220,7 +220,7 @@ def sync_historic_table(mssql_conn, config, catalog_entry, state, columns, strea
                                 , null _sdc_lsn_deleted_at
                                 , '00000000000000000000' _sdc_lsn_value
                                 , '00000000000000000000' _sdc_lsn_seq_value
-                                , 1 as _sdc_lsn_operation
+                                , 2 as _sdc_lsn_operation

```

Comment:
Update of a mistake in picking the right `_sdc_lsn_operation` type for inserts. Detailed explanation here: https://github.com/s7clarke10/pipelinewise-tap-mssql/pull/10#issuecomment-1560313935